vterm.el

```diff
@@ -688,7 +685,7 @@ Argument EVENT process event."
         (kill-buffer buf))
     ))
 
-(defun vterm--text-scale-mode (&optional argv)
+(defun vterm--text-scale-mode (&optional _argv)
   "fix `line-number' height for scaled text"
```

Comment:
I know there's a lot of work to be done on docstrings (in general in the package)... Can you capitalize the "fix" word?